tsconfig.json

```diff
@@ -9,6 +9,7 @@
       "@radix-ui/rect": ["./packages/core/rect/src"],
       "@radix-ui/react-accessible-icon": ["./packages/react/accessible-icon/src"],
       "@radix-ui/react-accordion": ["./packages/react/accordion/src"],
+      "@radix-ui/react-alert-dialog": ["./packages/react/alert-dialog/src"],
```

Comment:
not sure what happened to this one, but it seems it was either never there, or disappeared in a bad merge at some point

Reply:
strange !